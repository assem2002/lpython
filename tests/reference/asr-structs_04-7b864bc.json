{
    "basename": "asr-structs_04-7b864bc",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/structs_04.py",
    "infile_hash": "5951c49d2d7f143bbe3d67b982770ceb6d709939eb2d5ed544888f16",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-structs_04-7b864bc.stderr",
    "stderr_hash": "4c41571acfa90c551ece87ea4f5bb429349152d0202131c9b82030b1",
    "returncode": 2
}