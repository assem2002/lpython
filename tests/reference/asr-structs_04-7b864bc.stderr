semantic error: S() got multiple values for argument 'x'
 --> tests/errors/structs_04.py:9:18
  |
9 |     s: S = S(24, x=2)
  |                  ^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
